sys/usb/usbus/usbus.c

```diff
@@ -217,6 +222,99 @@ static void _usbus_init_handlers(usbus_t *usbus)
     }
 }
 
+static void _usbus_transfer_urb_submit(usbus_endpoint_t *usbus_ep,
+                                       usbus_urb_t *urb)
+{
+    /* Maximum between the urb length and the endpoint maximum size */
+    size_t len = urb->len > usbus_ep->maxpacketsize ?
+                 usbus_ep->maxpacketsize :
+                 urb->len;
+    usbdev_ep_xmit(usbus_ep->ep, urb->buf, len);
+    urb->buf += len;
+    urb->len -= len;
+}
+
+static void _usbus_transfer_complete(usbus_t *usbus, usbdev_ep_t *ep, usbus_handler_t *handler)
+{
+    usbus_endpoint_t *usbus_ep = _usbus_ep_from_usbdev(usbus, ep);
+    if (!clist_is_empty(&usbus_ep->urb_list)) {
```

Comment:
I agree with @OlegHahm.
Currently only `cdc_ecm` makes use of URB, why should boards that only use `cdc_acm` e.g. for stdio pay the price?
    
```suggestion
    if (IS_USED(MODULE_USBUS_URB) && !clist_is_empty(&usbus_ep->urb_list)) {
```

and

```make
ifneq (,$(filter usbus_cdc_ecm,$(USEMODULE)))
  USEMODULE += usbus_urb
endif
```